Resources/TRACKER.md

```diff
@@ -0,0 +1,168 @@
+
+# Clickstream Health Tracking!
+
+This is an additional feature of Clickstream SDK, which let us monitor the SDK's health i.e.
+
+ 1. **Drop rate of events:-** Number of events send by client app vs number of events successfully sent by SDK to backend. For example if 1000 events send by client app and 999 events are sent by Clickstream SDK to backend, then the drop rate is 0.1%.
+ 2. **Socket & connection error:-** We also track various other errors happening in the SDK like socket not connected, unable to send events due to low battery or network unavailable or Parsing exceptions thrown by backend.
+
+
+## Cocoapods
+
+To integrate Clickstream health tracker into your project specify the following in Podfile
+
+    pod 'Clickstream/Tracker'
+
+## Usage
+
+### Initialization
+Once intialized the tracker will start moniotiring the socket connections and funnel events in Clickstream, no explicit action or trigger needs to be done from client side.
+
+    let customerInfo = CSCustomerInfo(signedUpCountry: "India", email: "test@test.com", currentCountry: "91", identity: 105)
+    let sessionInfo = CSSessionInfo(sessionId: "1001")
+    let appInfo = CSAppInfo(version: "1.0")
+    let commonProperties = CSCommonProperties(customer: customerInfo, session: sessionInfo, app: appInfo)
+    let configs = ClickstreamHealthConfigurations(minimumTrackedVersion: "0.1", trackedVia: .internal)
+            
+    self.clickstream?.setTracker(configs: configs, commonProperties: commonProperties, dataSource: self)
+
+### ClickstreamHealthConfigurations
+
+Holds the configurations for health tracking, these onfigurations helps in having more control on the behaviour of Tracker.
+|Variable|Description|Type|Default Value|
+|--|--|-- |--|
+| minimumTrackedVersion |Track CS SDK health from minimum app version |String|-|
+| randomisingUserIdRemainders |Enable tracking for userId with following randomising remainder |[Int32]|[]|
+| trackedVia |Enable tracking via internal, external or both |TrackedVia|-|
+| verbosityLevel |Various verbosity levels |VerbosityLevel| VerbosityLevel.minimum|
```

Comment:
@AbhijeetMallick should we remove this mas?

Reply:
Yes,